File: examples/python/src/01_flowgraph.py
import json
from client import McpSseClient, env

def main():
    c = McpSseClient(env("MCP_BASE_URL"))
    res = c.call_tool("getFlowGraph", {"flowType": "all"})
    print(json.dumps(res, indent=2))

if __name__ == "__main__":
    main()

File: examples/node/src/01_flowgraph.ts
import { createClient } from "./_mcp_sse_client.js";

async function main() {
  const client = createClient();
  const res = await client.callTool("getFlowGraph", { flowType: "all" });
  console.log(JSON.stringify(res, null, 2));
  client.close();
}

main().catch((e) => {
  console.error(e);
  process.exit(1);
});
